yarn.lock

```diff
@@ -1470,24 +1470,6 @@
   resolved "https://registry.yarnpkg.com/@fluentui-react-native/design-tokens-windows/-/design-tokens-windows-0.42.0.tgz#6b7ca87ed30a92634e72cbe8fc1280fd21db50b8"
   integrity sha512-hBPCikSDJ3Rgw+mAbTgnNB83dn3SeSKPo4XNli22tbV08fN8P5mEaEg8Uw0cbVMN3D138ykNeAaK/lZjO9KIqA==
 
-"@fluentui-react-native/framework@0.8.31":
```

Comment:
I think I already accidentally fixed this in #2554?

https://github.com/microsoft/fluentui-react-native/pull/2554/files#diff-51e4f558fae534656963876761c95b83b6ef5da5103c4adef6768219ed76c2de

Reply:
Looks like you did!